vllm/executor/ray_gpu_executor.py

```diff
@@ -221,9 +236,24 @@ def _driver_execute_model(
         Passing None will cause the driver to stop the model execution
         loop running in each of the remote workers.
         """
+        assert not USE_SPMD_WORKER, (
+            "driver_worker does not exist for VLLM_USE_SPMD_WORKER=1")
         return self.driver_worker.execute_method("execute_model",
                                                  execute_model_req)
 
+    def execute_model(
+            self,
+            execute_model_req: ExecuteModelRequest) -> List[SamplerOutput]:
+        if not USE_SPMD_WORKER:
+            return super().execute_model(execute_model_req)
```

Comment:
this should be an attribute of the ray gpu executor, instead of reading a global variable.

Reply:
Done.